docs/reference/migration/migrate_2_0.asciidoc

```diff
@@ -138,6 +138,9 @@ In addition some query builders have been removed or renamed:
 * `filtered(...)` removed. Use `filteredQuery(...)` instead.
 * `inQuery(...)` removed.
 
+Also QueryBuilders.WrapperQueryBuilder(byte[] source, int offset, int length) has been removed, 
```

Comment:
the method names start with a lowercase `w`. Shouldn't we have both changes here? QueryBuilders and WrapperQueryBuilder constructor? those are both public.